app/models/genre_term.rb

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017, University of Alberta Libraries.
```

Comment:
I am following the upstream convention, where every file has a copyright notice.